src/handlers/assign.rs

```diff
@@ -487,6 +487,22 @@ pub(super) async fn handle_command(
                     name.to_string()
                 } else {
                     let teams = crate::team_data::teams(&ctx.github).await?;
+                    // Determine if assignee is a team. If yes, add the corresponding label
+                    // team name here is without prefix 't-' (e.g. 'compiler', 'libs', etc.)
+                    if let Some(team) = teams.teams.get(&name) {
+                        let t_label = format!("t-{}", team.clone().name);
+                        if let Err(err) = issue
+                            .add_labels(&ctx.github, vec![github::Label { name: t_label }])
+                            .await
+                        {
+                            if err.to_string().starts_with("Unknown") {
```

Comment:
I feel a little uncomfortable using string-based matching. I think the following might be a little more reliable:

```suggestion
                            if let Some(UnknownLabels { .. }) = err.downcast_ref() {
```